modules/nf-core/stranger/tests/nextflow.config

```diff
@@ -0,0 +1,5 @@
+process {
```

Comment:
Do you need this now?

Reply:
Otherwise the input vcf and output will be named the same.